main.cpp

```diff
@@ -443,11 +538,20 @@ struct LandingGear
     int numWheels = 8;
     float weightOfNose = 34.2f;
     int numMainLandingGear;
-    float pressureTire = 200.1f;
+    float maxTirePressure = 200.1f;
 
     int reduceLandingImpact (float tirePressure = 250.2f, int landingSpeed = 20);
     bool preventFuselageHittingGround (bool retractLandingGear = false);
     void toggleLandingGear (bool, std::string);
+    void inflate(float currentPressure)
+    {  
+        while  ( currentPressure <= maxTirePressure)
+        {
+            std::cout << "Tire pressure should ideally reach " << maxTirePressure << std::endl;
+            break;
+        }
```

Comment:
remove the `break`, because it prevents the `while` loop from looping.
You need to do something to make the conditional return `false` eventually.
Increment `currentPressure` in this loop.
- [ ] revise